sortinghat/core/schema.py

```diff
@@ -121,6 +135,10 @@ class ProfileInputType(graphene.InputObjectType):
     country_code = graphene.String(required=False)
 
 
+class OrganizationFilterType(graphene.InputObjectType):
```

Comment:
This filter should go in a different PR.

Reply:
OK, I am already preparing the PR with these changes.